src/QsCompiler/QirGeneration/README.md

```diff
@@ -0,0 +1,64 @@
+# QIR Emission - Preview Feature
+
```

Comment:
If not explained earlier, you may want to explain in this file 
* what the current implementation is (without QIR)
* what problems are there with the current implementation
* what solution is being worked on, what QIR stands for and how it solves the current problems